nodes/scheduling/nodes-scheduler-default.adoc

```diff
@@ -42,7 +42,7 @@ spec:
   defaultNodeSelector: type=user-node,region=east <2>
 ----
 <1> You can specify the name of a custom scheduler policy file.
-<2> Optionally, specify a default node selector to restrict pod placement to specific nodes.
+<2> Optionally, specify a default node selector to restrict pod placement to specific nodes. The default node selector is applied to the pods created in all namespaces. Pods can be scheduled on nodes with labels that match the default node selector and any existing pod node selectors. Namespaces having project-wide node selectors are not impacted even if this field is set. 
```

Comment:
It might be out of scope for this PR since it's existing content (your call! :slightly_smiling_face: ) but suggested change: 

<2> Optional: Specify